docs/integration/kubernetes.md

```diff
@@ -780,6 +780,24 @@ Container and Kubernetes metrics are collected along with labels and tags, which
 |----|----|-------------------------------------|
 |scheduler.attempts.total_num|uint64| number of attempts to schedule pods |
 
+### Runtime Metrics
+| Metric Name                            |Unit| Description                       |
+|----------------------------------------|----|-----------------------------------|
+| kubernetes.runtime.os_threads          |uint64| number of operating system threads used |
+| kubernetes.runtime.goroutines          |uint64| number of goroutines active  |
+| kubernetes.runtime.resident_memory     |uint64| amount of memory for process's code and data occupied |
```

Comment:
process's ==> process'

Reply:
commited to master, the commit does not show up in this PR by some reason